action.yml

```
@@ -194,6 +194,32 @@ inputs:
     required: false
     default: "true"
 
+  # TypeScript
+
+  tsc:
+    description: Enable or disable TypeScript checks
+    required: false
+    default: "false"
+  tsc_args:
+    description: Additional arguments to pass to the linter
+    required: false
+    default: ""
+  tsc_dir:
+    description: Directory where the TSC command should be run
+    required: false
+  tsc_extensions:
+    description: Extensions of files to check with TSC
+    required: false
+    default: "js"
+  tsc_command_prefix:
+    description: Shell command to prepend to the linter command. Will default to `npx --no-install` for NPM and `yarn run --silent` for Yarn.
+    required: false
+    default: ""
+  tsc_auto_fix:
+    description: Whether this linter should try to fix code style issues automatically. If set to `true`, it will only work if "auto_fix" is set to `true` as well
+    required: false
+    default: "true"
```

```suggestion
    default: "false"
```

Should be false since the linter doesn't support auto-fixing.